ios/Classes/GeofencingPlugin.m

```diff
@@ -1,7 +1,3 @@
-// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
Please leave the copyright as-is.

Reply:
I thought it was included by accident; will fix.